src/main/java/com/example/User.java

```diff
@@ -0,0 +1,9 @@
+package com.example;
+
+public interface User {
```

Comment:
`User` should be a concrete class, there's no need for this to be an interface as users will have _data_, rather than behaviour.